wheeledgraphicsview.cpp

```diff
@@ -0,0 +1,27 @@
+#include "wheeledgraphicsview.h"
+#include "mainwindow.h"
+
+#include <QGraphicsView>
+#include <QWheelEvent>
+
+
+
+WheeledGraphicsView::WheeledGraphicsView(QWidget *parent) : QGraphicsView(parent){
+}
+
+
+void WheeledGraphicsView::wheelEvent(QWheelEvent *event){
+
```

Comment:
indentation level is wrong. this should have single indentation, which is 4 spaces.